packages/aws-cdk-lib/aws-synthetics/lib/canary.ts

```diff
@@ -229,6 +230,51 @@ export interface CanaryProps {
    * @default - no rules applied to the generated bucket.
    */
   readonly artifactsBucketLifecycleRules?: Array<s3.LifecycleRule>;
+
+  /**
+   * Canary Artifacts in S3 encryption configuration.
+   * Artifact encryption is only supported for Node.js runtime.
+   *
+   * @default - Artifacts are encrypted at rest using an AWS managed key
+   *
+   * @see https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_artifact_encryption.html
+   */
+  readonly artifactS3Encryption?: ArtifactS3Encryption;
```

Comment:
Thank you. Canary Construct is implemented to flatten other properties as well, so I agree that it would be better to flatten it in this case as you suggested. I will modify it to make it flat.​​​​​​​​​​​​​​​​

Reply:
I've changed it to flat.